frontend/src/bundles/studio/pages/studio.tsx

```diff
@@ -98,12 +110,22 @@ const Studio: React.FC = () => {
                     />
                 }
                 right={
-                    <Button
-                        variant="primaryOutlined"
-                        label="Submit"
-                        sx={{ width: '100px' }}
-                        onClick={handleSubmit}
-                    />
+                    <Flex gap="10px">
+                        <LibraryInput
+                            defaultValue={videoName}
+                            className={styles['videoName']}
+                            variant="unstyled"
+                            placeholder="Untitled video"
+                            onBlur={handleEditVideoName}
+                        />
+                        <Button
+                            variant="primaryOutlined"
+                            label="Submit"
+                            sx={{ width: '100px' }}
```

Comment:
```suggestion
                            width='100px'
```
Since `Remove sx prop` PR we can pass Chakra properties to our custom button

Reply:
Actually that pr removed this sx prop (I looked it up), but we have it again in next and I don't know why

Reply:
Maybe, it could happen due to resolving merge conflicts
https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/218/files#r1751613011